runtime/decimal.c

```diff
@@ -9,6 +9,10 @@ typedef GC decQuad *DecimalPtr;
 // XXX need to investigate DEC_Underflow
 #define DECIMAL_STATUS_FAIL DEC_Errors
 
+#define STR_CONVERT(x) #x
+#define STR(x) STR_CONVERT(x)
+#define INT64_MAX_LEN sizeof(STR(INT64_MIN))
```

Comment:
Move these defines to just before you use INT64_MAX_LEN.

Reply:
Changed